GOVERNANCE.md

```diff
@@ -0,0 +1,122 @@
+# Governance
+
+## Glossary
+
+* **Advisory Committee.**  A group of people with representation from a variety of AMP's constituencies including Users, End-users and Collaborators who provide advice to the Technical Steering Committee.
+
+* **Collaborators.**  People who have been granted write-access to ampproject repositories.
+
+* **Contributors**.  People who have opened a pull request or an issue, or commented on an issue. Contributors are granted certain permissions on the repositories to make it easier to participate--e.g. assigning issues, attaching labels to issues, etc.
+
+* **Users.** Developers who use AMP but haven’t contributed to it (yet).
+
+* **End-users**. People who consume content distributed using the AMP format.
+
+* **Owners**.  People who are most familiar with a particular set of code and who have been granted the power (by Owners at a higher level) for approving changes to that code.  AMP will use a system similar to [Chromium's OWNERS](https://chromium.googlesource.com/chromium/src/+/master/docs/code_reviews.md#OWNERS-files) system.  The AMP Project will follow Chromium's [expectation for owners](https://chromium.googlesource.com/chromium/src/+/master/docs/code_reviews.md#expectations-of-owners).
```

Comment:
@cramforce can we clarify here that owners will be a "github collaborator".

also my question earlier in hangout, do collaborators also get "master merge" access

my current implementation is that, they do not have "master merge" access and that they only turn a status check to green and a "collaborator" (one who has approval status) merges the PR for them if they are not a "collaborator"

Reply:
> also my question earlier in hangout, do collaborators also get "master merge" access

No they shouldn't.

> my current implementation is that, they do not have "master merge" access and that they only turn a status check to green and a "collaborator" (one who has approver status) merges the PR for them if they are not a "collaborator"

Yeah, that sounds good.

These are technicalities, however, which you might want to be able to change as needed without having to go through a governance document modification. We want to make sure not too specify this in too much details here.